opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -395,40 +394,27 @@ message SummaryValue {
 
   // The total number of recorded values since start_time. Optional since
   // some systems don't expose this.
-  google.protobuf.Int64Value count = 3;
+  int64 count = 3;
 
   // The total sum of recorded values since start_time. Optional since some
   // systems don't expose this. If count is zero then this field must be zero.
-  // This field must be unset if the sum is not available.
-  google.protobuf.DoubleValue sum = 4;
-
-  // The values in this message can be reset at arbitrary unknown times, with
-  // the requirement that all of them are reset at the same time.
-  message Snapshot {
-    // The number of values in the snapshot. Optional since some systems don't
-    // expose this.
-    google.protobuf.Int64Value count = 1;
-
-    // The sum of values in the snapshot. Optional since some systems don't
-    // expose this. If count is zero then this field must be zero or not set
-    // (if not supported).
-    google.protobuf.DoubleValue sum = 2;
-
-    // Represents the value at a given percentile of a distribution.
-    message ValueAtPercentile {
-      // The percentile of a distribution. Must be in the interval
-      // (0.0, 100.0].
-      double percentile = 1;
-
-      // The value at the given percentile of a distribution.
-      double value = 2;
-    }
+  double sum = 4;
+
+  // count_and_sum_unspecified must be set to true if count and sum values are unknown or
+  // unspecified.
+  bool count_and_sum_unspecified = 5;
```

Comment:
Please split this into 2:
1. Removes the Snapshot -> makes it closer to openmetrics
2. Remove usage of the wrapped values.

Reply:
Done. This is part 1.